content/docs/guides/browser-automation.md

```diff
@@ -120,28 +120,36 @@ class ExampleSteps: En {
 ```
 
 ```javascript
-var driver = new webdriver.Builder().build();
-driver.get('http://www.google.com');
+const { Given, When, Then, AfterAll } = require('cucumber');
+const { Builder, By, Capabilities, Key } = require('selenium-webdriver');
+const { expect } = require('chai');
 
-var element = driver.findElement(webdriver.By.name('q'));
-element.sendKeys('Cheese!');
-element.submit();
+require("chromedriver");
 
-driver.getTitle().then(function(title) {
-  console.log('Page title is: ' + title);
+// driver setup
+const capabilities = Capabilities.chrome();
+capabilities.set('chromeOptions', { "w3c": false });
+const driver = new Builder().withCapabilities(capabilities).build();
+
+Given('I am on the Google search page', async function () {
+    await driver.get('http://www.google.com');
 });
 
-driver.wait(function() {
-  return driver.getTitle().then(function(title) {
-    return title.toLowerCase().lastIndexOf('cheese!', 0) === 0;
-  });
-}, 3000);
+When('I search for {string}', async function (searchKey) {
```

Comment:
lets call this searchTerm, not to confuse with the key.return you're using below

Reply:
Thanks, Done.